mobile/library/objective-c/BUILD

```diff
@@ -1,16 +1,16 @@
-load("@envoy//bazel:envoy_build_system.bzl", "envoy_mobile_copts")
+load("//bazel:apple.bzl", "envoy_objc_library")
 
 licenses(["notice"])  # Apache 2
 
 exports_files([
     "EnvoyEngine.h",
 ])
 
-objc_library(
+envoy_objc_library(
     name = "envoy_engine_objc_lib",
     srcs = [
         "EnvoyConfiguration.h",
-        "EnvoyConfiguration.m",
+        "EnvoyConfiguration.mm",
```

Comment:
Out of curiosity what's the difference between the ".m" and ".mm" extension?

Reply:
`.m` is Objective-C. `.mm` is Objective-C++.

Reply:
Oh, of course! Thanks.